Bri's Assignment Submissions

Briadean opened the pull request:
Comprehensions Assignment
Generator Activity
Iterator Activity

Commit:
For Lesson 01:
Comprehensions Assignment
Generator Activity
Iterator Activity

Commit:
Lesson02 Generators and Closures Assignment

Commit:
Merge branch 'master' of https://github.com/UWPCE-PythonCert-ClassRepos/SP2018-Python220-Accelerated

Commit:
Completed Lesson03 Factorial Assignment
Bulk of Lesson07 Relational DB Activity- 1 todo, 1 bug to fix

Commit:
Lesson 03 Locke Exercise commit- still have an error to fix

Commit:
Lesson 07 Activity - need to figure out how to make
pprint module work with sqlite query

Commit:
Fixed issues with lesson01 Prime function

Commit:
Completed lesson05 Assignment

brydavis reviewed (commented):
for `def __enter__`, must return `self`

Commit:
Fixed Locke assignment error

Commit:
Completed Lesson05 Activity

Commit:
Completed Lesson06 Assignment

Commit:
Completed lesson06 Activity

Commit:
Lesson 07 Assignment (I got it to populate a DB, but chose to
try and use unique keys and its been a bear. Its broken, but i feel
i've sunk too much time into it to try and go back at this point.)

Lesson09 Assignment (Also still broken, but it is Multithreaded!
Still trying to figure out how to make it only return the information
for the articles that contain the specified keyword instead of all of
top articles for the day)

Commit:
Lesson07 Assignment schema

Commit:
Completed Lesson10 Activity demo for python's line
by line memory profiler.
